lib/reference/properties.js

```diff
@@ -311,55 +320,70 @@ const longhandSubPropertiesOfShorthandProperties = new Map([
 			'mask-origin',
 			'mask-clip',
 			'mask-composite',
-		]),
+		],
 	],
 	[
 		'outline',
-		new Set([
+		[
 			// prettier-ignore
 			'outline-color',
 			'outline-style',
 			'outline-width',
-		]),
+		],
 	],
 	[
 		'padding',
-		new Set([
+		[
 			// prettier-ignore
 			'padding-top',
+			'padding-right',
 			'padding-bottom',
 			'padding-left',
-			'padding-right',
-		]),
+		],
 	],
 	[
 		'text-decoration',
-		new Set([
+		[
 			// prettier-ignore
 			'text-decoration-color',
 			'text-decoration-style',
 			'text-decoration-line',
-		]),
+		],
 	],
 	[
 		'text-emphasis',
-		new Set([
+		[
 			// prettier-ignore
 			'text-emphasis-style',
 			'text-emphasis-color',
-		]),
+		],
 	],
 	[
 		'transition',
-		new Set([
-			'transition-delay',
-			'transition-duration',
+		[
+			// prettier-ignore
 			'transition-property',
+			'transition-duration',
 			'transition-timing-function',
-		]),
```

Comment:
I *think* this is the only one that had the "wrong" order - see the CSS spec for `single-transition` (ref: [CSS spec](https://w3c.github.io/csswg-drafts/css-transitions/#single-transition)) for the right order.

Reply:
The examples on https://developer.mozilla.org/en-US/docs/Web/CSS/transition#syntax seems to indicate that
`name | duration | easing function` and `name | duration | delay` are both valid.

Reply:
Yeah, I think the general order is agnostic (that's what the [double bar means](https://developer.mozilla.org/en-US/docs/Web/CSS/Value_definition_syntax#double_bar)), moreso that the *first* time value is parsed as a duration, and the *second* is a delay; from your link,

> The first value that can be parsed as a time is assigned to the [transition-duration](https://developer.mozilla.org/en-US/docs/Web/CSS/transition-duration), and the second value that can be parsed as a time is assigned to [transition-delay](https://developer.mozilla.org/en-US/docs/Web/CSS/transition-delay).